### ISSUE TITLE
can the lib support the polygon with holes?

### ISSUE THREAD

**Issue by dyh333** [maptime-ams/geojson-3d#4]
hi, @maptime-ams.  as the title, thanks.

**Comment by bertspaan** [comment]
Yes, it can; but currently, it doesn't. Only polygons with a [single LinearRing](http://geojson.org/geojson-spec.html#polygon) are supported, see [line 63 of `js/geo.js`](https://github.com/maptime-ams/geojson-3d/blob/gh-pages/js/geo.js#L63). However, it should be easy to add support for other GeoJSON geometry types (like LineStrings, MultiPolygons, and Polygons with holes).
